utils/tracing/go/tracing.go

```diff
@@ -120,3 +123,15 @@ func InitCustomTracer(url string, traceRate float64, logger *log.Logger, attr ..
 	f := initTracer(tp)
 	return f, nil
 }
+
+// GrpcInterceptorServer returns a grpc server instrumented with an opentelemetry interceptor
+// which enables tracing of grpc requests.
+func GrpcInterceptorServer() *grpc.Server {
+	return grpc.NewServer(grpc.UnaryInterceptor(otelgrpc.UnaryServerInterceptor()))
+}
+
+// GrpcInterceptorDial creates a connection to the provided address, which is instrumented with
+// an opentelemetry client interceptor enabling the tracing to client grpc messages.
+func GrpcInterceptorDial(addr string) (*grpc.ClientConn, error) {
```

Comment:
```suggestion
func DialGRPCWithUnaryInterceptor(addr string) (*grpc.ClientConn, <one more arg>,error) {
```

pass `grpc.WithInsecure()` as an additional argument